### PR TITLE
Fix segfault in Cobyla::ComputeObjectiveAndConstraint

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: lib/src/Base/Optim/Cobyla.cxx [hunk 1]
```diff
@@ -219,7 +219,7 @@ int Cobyla::ComputeObjectiveAndConstraint(int n,
 
   /* Convert the input vector in OpenTURNS format */
   Point inPoint(n);
-  memcpy(&inPoint[0], &x[0], n * sizeof(Scalar));
+  memcpy(&inPoint[0], x, n * sizeof(Scalar));
```
Comment (dbarbier) [inline lib/src/Base/Optim/Cobyla.cxx]: Why is this change needed?

Comment (jschueller) [reply lib/src/Base/Optim/Cobyla.cxx]: Not needed, just simpler.

Comment (dbarbier) [reply lib/src/Base/Optim/Cobyla.cxx]: ok

**Comment by regislebrun** [comment]
+1
